src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1792,6 +1821,150 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly mutationEnrichmentProfiles = remoteData<MolecularProfile[]>({
+        await: () => [
+            this.molecularProfilesInStudies
+        ],
+        invoke: async () => {
+            const profiles: MolecularProfile[] = _.filter(this.molecularProfilesInStudies.result, 
```

Comment:
another nitpick, but _.find return first matching item

Reply:
I need both the first item (to set as default) and all filtered items (to return from the function) in this function.

Reply:
gotcha. so per the note below, i think the default needs to be a computation of this, or we should set it in onResult.  probably a computation which only ever sets it once is best.  i don't know the exact timing/lifecycle of onResult